calico-enterprise/reference/resources/webhook.mdx

```diff
@@ -0,0 +1,151 @@
+| name         | Configuration variable name.                                               | string                                                                      | yes                                 |
+| value        | Direct value for the variable.                                             | string                                                                      | yes if `valueFrom` is not specifed  |
+| valueFrom    | Value defined either in a Kubernetes ConfigMap or in a Kubernetesa Secret. | [SecurityEventWebhookConfigVarSource](#securityeventwebhookconfigvarsource) | yes if `value` is not specified     |
+
+### SecurityEventWebhookConfigVarSource
+
+| Field            | Description                       | Schema                                                                                                       | Required                                  |
+| ---------------- | --------------------------------- | ------------------------------------------------------------------------------------------------------------ | ----------------------------------------- |
+| configMapKeyRef  | Kubernetes ConfigMap reference.   | `ConfigMapKeySelector` (referenced ConfigMap key should exist in the `tigera-intrusion-detection` namespace) | yes if `secretKeyRef` is not specified    |
+| secretKeyRef     | Kubernetes Secret reference.      | `SecretKeySelector` (referenced Secret key should exist in the `tigera-intrusion-detection` namespace)       | yes if `configMapKeyRef` is not specifed  |
+
+### Status
+
+Field `status` reflects the health of a webhook. It is a list of [Kubernetes Conditions](https://pkg.go.dev/k8s.io/apimachinery@v0.23.0/pkg/apis/meta/v1#Condition).
+
+## Query
+
+SecurityEventWebhooks use a domain-specific query language to select which records
+from the data set should trigger the HTTP request.
+
+The query language is composed of any number of selectors, combined
+with boolean expressions (`AND`, `OR`, and `NOT`), set expressions
+(`IN` and `NOTIN`) and bracketed subexpressions. These are translated
+by {{prodname}} to Elastic DSL queries that are executed on the backend.
+
+Set expressions support wildcard operators asterisk (`*`) and question mark (`?`).
+The asterisk sign matches zero or more characters and the question mark matches a single character.
+Set values can be embedded into the query string or reference the values
+in the global alert substitution list.
+
+A selector consists of a key, comparator, and value. Keys and values
+may be identifiers consisting of alphanumerics and underscores (`_`)
+with the first character being alphabetic or an underscore, or may be
+quoted strings. Values may also be integer or floating point numbers.
+Comparators may be `=` (equal), `!=` (not equal), `<` (less than),
+`<=` (less than or equal), `>` (greater than), or `>=` (greater than
+or equal).
+
+Keys must be indexed fields in their corresponding data set. See the
```

Comment:
This paragraph makes no sense here (I assume it was taken from https://docs.tigera.io/calico-enterprise/3.16/reference/resources/globalalert#query). In this case the only index we deal with is the events index (there is no appendix in this page :wink:). The source of truth for the fields we validate is in https://github.com/tigera/calico-private/blob/master/libcalico-go/lib/validator/v3/query/validate_events.go#L12 but I don't think we want to go in that level of details here.

Reply:
Yeah, the Query section was copied. I have missed a couple of references - removed now.